environment-pipelines/buildspec-apply.yml

```diff
@@ -17,6 +17,32 @@ phases:
       - echo -e "\nWorking on environment ${ENVIRONMENT}"
       - cd "terraform/environments/${ENVIRONMENT}"
       - terraform apply plan.tfplan
+      - echo -e "\nGenerating manifests and deploying AWS Copilot environment resources"
+      - |
+          if [[ "${COPILOT_PROFILE}" != *"prod"* ]]
+          then
+            cd "${CODEBUILD_SRC_DIR}"
+            platform-helper environment generate --name "${ENVIRONMENT}"
+            copilot env init --name "${ENVIRONMENT}" --profile "${COPILOT_PROFILE}" --default-config
+            platform-helper copilot make-addons
```

Comment:
Do we need this here? It was previously in the environment pipeline but I think it only affects service level addons?

Reply:
I don't think we need it. I think you're right that in terraform world, it's only going to be creating svc addons

Reply:
Agree, service addons are still added to version control.
```suggestion
```